src/lib/theme.ts

```diff
@@ -100,5 +100,5 @@ export const switchTheme = (name: ThemeName) => {
 };
 
 export const applyTheme = (theme: Theme) => {
-    return Object.entries(theme).reduce((s, [n, v]) => `${s}${n}:${v};`, '');
+    return Object.entries(theme.vars).reduce((s, [n, v]) => `${s}${n}:${v};`, '');
```

Comment:
Just a suggestion for clearer paramater names:
```suggestion
    return Object.entries(theme.vars).reduce((rules, [prop, val]) => `${rules}${prop}:${val};`, '');
```